src/tests/topp/QualityControl_1_out.mzTab

```diff
@@ -4,18 +4,23 @@ MTD	mzTab-type	Quantification
 MTD	title	ConsensusMap export from OpenMS
 MTD	description	OpenMS export from consensusXML
 MTD	protein_search_engine_score[1]	[, , one-peptide-rule XTandem, ]
+MTD	protein_search_engine_score[2]	[, , one-peptide-rule XTandem, ]
```

Comment:
why are there 3?

Reply:
So yes, this is kind of a Todo and made me think if it wouldn't be nicer to have run level info. But yes, since for now I make peptide search scores unique, we should probably do the same for proteins.

Reply:
yes I think so but this is currently not supported by mzTab